### DISCUSSION TITLE
请问 ORM 框架上有考虑集成 Prisma 吗

### DISCUSSION THREAD

**Discussion by 7kyun** [midwayjs/midway#3812] [General / 闲聊]
从下载量以及 star 数来看，如今 Prisma 已经超过 TypeORM，考虑 TypeORM 的更新频率和各种奇怪的坑，后续新项目想尝试下采用Prisma 进行开发看看

**Comment by czy88840616** [comment]
Prisma 的使用目前不需要框架做什么的。。

**Reply by 7kyun** [reply]
后面抽空看了下，确实不需要，嘿嘿 ♪( ´▽｀)

**Reply by youwei997** [reply]
prisma时区的问题都没解决

**Comment by cvpv** [comment]
我目前也在基于兴趣的学习midway。

请问目前最合适的orm框架是什么？目前看typeorm 似乎有很多争议
https://github.com/typeorm/typeorm/issues/3267
https://github.com/Kononnable/typeorm-model-generator/issues/329

**Reply by czy88840616** [reply]
适合自己的才是最好的，不做特别推荐，比如我自己就只用 sequelize。

**Reply by cvpv** [reply]
> 适合自己的才是最好的，不做特别推荐，比如我自己就只用 sequelize。

非常感谢您，我上周开始准备在我的项目内集成TypeORM。结果踩了很多坑，我也决定切回sequelize...
